app/Models/FeedDAO.php

```diff
@@ -421,4 +423,48 @@ public static function daoToFeed($listDAO, $catID = null) {
 
 		return $list;
 	}
+
+	private function addColumn($name) {
+		Minz_Log::warning('FreshRSS_FeedDAO::addColumn: ' . $name);
+		$hasTransaction = false;
+		try {
+			$stm = null;
+			if ('muteStrategy' === $name) {	//v1.10
+				if (!$this->bd->inTransaction()) {
+					$this->bd->beginTransaction();
+					$hasTransaction = true;
+				}
+				$stm = $this->bd->prepare('ALTER TABLE `' . $this->prefix . 'feed` ADD COLUMN `muteStrategy` INT(11) NOT NULL DEFAULT 0');
+				if ($stm && $stm->execute()) {
+					if ($hasTransaction) {
+						$this->bd->commit();
+					}
+					return true;
+				}
+				if ($hasTransaction) {
+					$this->bd->rollBack();
+				}
+			}
+		} catch (Exception $e) {
+			Minz_Log::error('FreshRSS_FeedDAO::addColumn error: ' . $e->getMessage());
+			if ($hasTransaction) {
+				$this->bd->rollBack();
+			}
+		}
+		return false;
+	}
+
+	public function autoUpdateDb($errorInfo) {
```

Comment:
We need to add the code for SQLite and PostgreSQL too, in FeedDAOSQLite.php and FeedDAOPGSQL.php.
Examples for the entry table:
* https://github.com/FreshRSS/FreshRSS/blob/f632a346269100d6a93bef318ffa66c97f16f6fa/app/Models/EntryDAOSQLite.php#L9-L26
* https://github.com/FreshRSS/FreshRSS/blob/f632a346269100d6a93bef318ffa66c97f16f6fa/app/Models/EntryDAOPGSQL.php

Since the code is similar for all tables, we could consider factorising it somehow.

As it is the first new field after we have introduced PostgreSQL, I do not yet have the full example of it.

Reply:
Will do